test/unittests/test_end2end.py

```
@@ -22,6 +22,16 @@
 
 from ovos_bus_client.session import SessionManager, Session
 
+
+def _simple_plugin_available() -> bool:
+    try:
+        from ovos_plugin_manager.audio import find_audio_service_plugins
+        return "ovos_simple" in find_audio_service_plugins()
+    except Exception:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)